queue_services/entity-emailer/src/entity_emailer/email_templates/NR-REFUND.html

```diff
@@ -1,66 +1,17 @@
-<!DOCTYPE html>
-<html lang="en">
-  <head>
-    <base href="/">
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="referrer" content="origin-when-cross-origin"/>
-    <meta name="author" content="BC Registries and Online Services">
-    <title>Name Request notification from the Business Registry</title>
-    [[style.html]]
-  </head>
+This is confirmation that your name request {{ nr_number }} has been cancelled and will not be reviewed by staff.
 
-  <body>
-    <table class="body-table" role="presentation">
-      <tr>
-        <td>
-          [[header.html]]
+**If you paid by credit card or a BC OnLine account:**
+A refund of **${{ refund_value }} CAD** (total payment value including service fee) will be applied to your original payment method. Refunds are usually processed within 10 to 14 business days.
 
-          <div class="container">
+**If you paid by Pre-authorized Debit (PAD):**
+If the name request was cancelled on the day of the original purchase and prior to 7:00 pm Pacific time, you will not be charged. If the NR was cancelled after 7:00 pm Pacific time on the day of the purchase, you will receive a refund to your PAD account within 10 to 14 business days.
 
-            [[whitespace-16px.html]]
-            <p>
-              This is confirmation that your name request {{ nr_number }} has been cancelled and will not be reviewed
-              by staff.
-            </p>
+**If you mailed payment to BC Registries Services:**
+You will not receive an automatic refund. Please call 1-877-370-1033 to request a refund.
 
-            [[whitespace-16px.html]]
-            <p><strong>If you paid by credit card or a BC OnLine account:</strong></p>
-            <p>
-              A refund of <strong>${{ refund_value }} CAD</strong> (total payment value
-              including service fee) will be applied to your original payment method. Refunds are usually processed
-              within 10 to 14 business days.
-            </p>
+Thank you for using Name Request. To submit another Name Request, visit [www.bcregistry.ca/namerequest](www.bcregistry.ca/namerequest)
 
-            [[whitespace-16px.html]]
-            <p><strong>If you paid by Pre-authorized Debit (PAD):</strong></p>
-            <p>
-              If the name request was cancelled on the day of the original purchase and prior to 7:00 pm Pacific
-              time, you will not be charged. If the NR was cancelled after 7:00 pm Pacific time on the day of the
-              purchase, you will receive a refund to your PAD account within 10 to 14 business days.
-            </p>
+Note: We are unable to notify you when a refund payment has been processed. If you have not received your
+refund after 14 business days please contact us.
```

Comment:
Should have a comma between "days" and "please".

(Reason: this is a if-then statement with no "then", so should have a comma instead.)